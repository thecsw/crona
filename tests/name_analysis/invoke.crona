f : int (arg1 : int, arg2 : bool){
}
g : void (){
	a : int;
	f(a, true);
	g();
}
