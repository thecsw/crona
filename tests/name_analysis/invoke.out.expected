f(int,bool->int):int(arg1(int):int, arg2(bool):bool){
}
g(->void):void(){
	a(int):int;
	f(int,bool->int)(a(int), true);
	g(->void)();
}